tests/org.eclipse.ui.tests.navigator/src/org/eclipse/ui/tests/navigator/resources/ResourceMgmtActionProviderTests.java

```diff
@@ -0,0 +1,221 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
```

Comment:
please stick to the format ` * Copyright (c) 2024 <your name or compan> and others.`

Reply:
I had hoped to avoid putting my name on this via the "more generic copyright header" option mentioned @ https://www.eclipse.org/projects/handbook/#ip-copyright-headers , but it's all good / change added.